app/services/katello/pulp3/repository/yum.rb

```diff
@@ -332,28 +343,64 @@ def copy_content_from_mapping(repo_id_map, options = {})
           multi_copy_units(repo_id_map, dependency_solving)
         end
 
-        def copy_content_for_source(source_repository, options = {}) # rubocop:disable Metrics/CyclomaticComplexity
-          filters = [ContentViewErratumFilter, ContentViewPackageGroupFilter, ContentViewPackageFilter].collect do |filter_class|
+        def errata_exclusion_helpers(errata_filters, source_repository_ids)
```

Comment:
I suppose I could make this methods returns slightly less odd and more reusable by returning `all_included_errata` and `all_excluded_errata`.

Reply:
oh i guess, this is kinda what my question above was getting at.  I think that would be simpler, to understand. 

The name of this method is also a bit confusing, maybe filtered_errata() or something?